content/guides/getstarted/setup-local-development.md

```diff
@@ -292,6 +323,17 @@ on your machine:
 solana-test-validator
 ```
 
+> In WSL you may need to first navigate to a folder where you have default write
+> access:
+>
+> ```shell
+> cd home
+> cd <your username>
```

Comment:
In unix system, the user can nagivate to their home directory using the `~` path:

```suggestion
> cd ~
```